base/src/main/java/com/smartdevicelink/proxy/rpc/WindowStatus.java

```diff
@@ -0,0 +1,54 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
+public class WindowStatus extends RPCStruct {
+
+    public static final String KEY_LOCATION = "location";
+    public static final String KEY_WINDOW_STATE = "state";
+
+    /**
+     * Constructs a newly allocated WindowStatus object
+     */
+    public WindowStatus() {}
+
+    /**
+     * Constructs a newly allocated WindowStatus object indicated by the Hashtable parameter
+     * @param hash The Hashtable to use
+     */
+    public WindowStatus(Hashtable<String, Object> hash) {
+        super(hash);
+    }
+    /**
+     * Constructs a newly allocated WindowStatus object
+     * @param location describes location of Window of Door/liftgate
+     * @param state describes state of Window of Door/liftgate
+     */
+    public WindowStatus(@NonNull Grid location, @NonNull WindowState state) {
+        this();
+        setLocation(location);
+        setWindowState(state);
+    }
+
+
```

Comment:
```suggestion
    /**
     * Sets the location.
     *
     * @param location
     */
```
Please include javadocs